### PR TITLE
Multimedia: adding fullscreen button

### PR DESCRIPTION
**Pull request by Garneauma**
Adding full screen button to multimedia video controls for HTML and YouTube videos.

Changes related to WET-414

### CODE REVIEW

File: src/i18n/i18n.csv [hunk 1]
```diff
@@ -77,6 +77,7 @@ Multimedia player,mmp-play,Play,Lire,Speel,Luaj,لعب,Խաղալ,Играя,播
...
 ,cc-err,Error loading closed captions,Erreur dans le chargement des sous-titres,Fout tydens die laai gesluit onderskrifte,Captions Gabim gjatë ngarkimit të mbyllura,خطأ في تحميل تعليق مغلق,Սխալի բեռնելիս փակ վերնագրեր,Грешка при зареждане на затворени надписи,載入字幕錯誤,加载字母错误,Chyba při načítání uzavřené titulky,Fout bij het tonen van de bijschriften,Viga subtiitrite avamisel,Fehler beim Laden Untertiteln,Σφάλμα κατά τη φόρτωση κλειστά λεζάντες,लोड करने में त्रुटि बंद कैप्शन,Hiba a felirat betöltése közben,Villa við að hlaða Skjátextar,Galat saat memuat subtitel,Errore caricamento sottotitoli,ᑕᒻᒪᕐᓂᖅᑕᖃᖅᑐᖅ ᑕᐃᒎᓯᖏᓐᓂ ᓴᕿᔮᖅᑎᑦᓯᓂᕐᒥ,クローズドキャプションを実行できませんでした。,로드하는 중 오류가 발생 자막,Kļūda lejuplādējot subtitrus,Klaida kraunant didžiųjų raidžių įjungimą,Błąd w ładowaniu podpisów,Ocorreu um erro no carregamento da legenda,Ocorreu um erro no carregamento da legenda,Ошибка загрузки субтитров,Chyba pri načítaní uzavreté titulkami,Error al cargar subtítulos,ข้อผิดพลาดคำอธิบายปิดโหลด,Kapalı eklenti yükleme hatası,Помилка завантаження субтитрів,Lỗi khi đang nạp chú thích đóng
+,fs,Enter full screen,Ouvrir plein écran,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (duboisp) [inline src/i18n/i18n.csv:80]: Did you also added this in our master spreadsheet? Then you can run a special grunt command to update our local i18n.csv file.

**Review by duboisp — CHANGES_REQUESTED** [review]
Either:
* Make it optional (no full screen button) unless if the author has explicitly configured the multimedia to show the full screen button
* Create a video controller (Play/Pause Volume...) for the full screen mode and show the full screen button by default

Check if we do watch and react on when the screen orientation change (Portrait / landscape ) in regard of full screen mode. If so, for example check for conformance to https://www.w3.org/WAI/WCAG22/Understanding/orientation.html

Note: For accessibility, the full screen content need to comply like the dialog when open.

File: src/i18n/i18n.csv [hunk 2]
```diff
@@ -212,7 +213,7 @@ Disable/enable WET plugins and polyfills (v4.x+),wb-disable,Switch to basic HTML
 ,disable-notice,You are viewing Basic HTML view. Some features may be disabled.,Vous naviguez présentement sur la version HTML simplifiée de cette page. Certaines fonctionnalités peuvent être désactivées.,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Dismissable content,dismiss,Dismiss,Écarter,,,,,,,,,,,,,,,,,,ᖁᔭᓈᕐᓗᒍ,,,,,,,,,,,,,,
 Filter,fltr-lbl,"Filter<span class=""wb-inv""> content: results appear below as you type.</span>","Filtrer<span class=""wb-inv""> le contenu: Les résultats s'afficherons au moment même de la saisie.</span>",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-,fltr-info,Showing <span data-nbitem></span> filtered from <span data-total></span> total entries,Affiche <span data-nbitem></span> de <span data-total></span> éléments filtrés.,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+,fltr-info,Showing _NBITEM_ filtered from _TOTAL_ total entries,Affiche _NBITEM_ de _TOTAL_ éléments filtrés.,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```
Comment (duboisp) [inline src/i18n/i18n.csv:216]: Est-ce que ce changement était intentionelle?

Probablement un problème de synchronization entre le fichier csv et le google spreadsheet.

File: src/i18n/i18n.csv [hunk 3]
```diff
@@ -59,7 +59,7 @@ Archived Web page template,arch-pg,This Web page has been archived on the Web.,C
...
-,tab,Tab #num#,Onglet #num#,Tab #num#,Tab #num#,#num# التبويب,ականջակալ #num#,Раздел #num#,标签 #num#,标签 #num#,Tab #num#,Lab #num#,Sakk #num#,Tab #num#,Αυτί #num#,टैब #num#,Tab #num#,Tab #num#,Tab #num#,Scheda #num#,ᓱᑯᑦᑎᐊᓅᖅᑕᐅᑦ #ᓈᓴᐅᑦ#,タブ #num#,탭 #num#,Tab #num#,Skirtukas #num#,Patka #num#,Aba #num#,Aba #num#,Вкладка #num#,Tab #num#,Pestaña #num#,แท็บ #num#,Sekme #num#,табуляція #num#,Chuyển hướng #num#
+,tab,Tab #num#,Onglet #num#,Tab #num#,Tab #num#,#num# التبويب,ականջակալ #num#,Раздел #num#,标签 #num#,标签 #num#,Tab #num#,Lab #num#,Sakk #num#,Tab #num#,Αυτί #num#,टैब #num#,Füllemez #num#,Tab #num#,Tab #num#,Scheda #num#,ᓱᑯᑦᑎᐊᓅᖅᑕᐅᑦ #ᓈᓴᐅᑦ#,タブ #num#,탭 #num#,Tab #num#,Skirtukas #num#,Patka #num#,Aba #num#,Aba #num#,Вкладка #num#,Tab #num#,Pestaña #num#,แท็บ #num#,Sekme #num#,табуляція #num#,Chuyển hướng #num#
```
Comment (duboisp) [inline src/i18n/i18n.csv:62]: Please revert this change

"Füllemez" to "Tab" car "Füllemez" veux dire "plaque d'oreille" ce qui n'est une traduction exacte dans ce contexte-ci.

**Review by duboisp — CHANGES_REQUESTED** [review]
Tested locally and reviewed the code, everything about adding the optional fullscreen button work.

But some additional change was observed from the i18n file which don't seems to be typo fixes. Can you fix it at the source in the google spreasheet and then regenerate the i18n csv file.

Thanks

**Review by duboisp — APPROVED** [review]
Change requested was addressed

**Comment by Christopher-O** [comment]
Just wanted to flag that in full screen for regular multimedia, the closed captions do not work whether you have them activated  before you open full screen, or when you are already in the full screen.  They do however work in YouTube, but only if you press the captions button before you go to full screen, otherwise in full screen there is no way to show the captions.

**Pull request merged**